tests/system/test_integration_acm.py

```diff
@@ -324,6 +324,28 @@ def test_backend_not_ok(self):
         assert r2.status_code == 200, r2.status_code
         assert r2.json() == {"transaction_sample_rate": "0.3"}, r2.json()
 
+    def test_rum_after_backend(self):
```

Comment:
nit: the previous issue would only be uncovered if the order is changed `test_backend_after_rum`.

Reply:
that's also exercised here if you are referring to that https://github.com/elastic/apm-server/pull/2747/files#diff-6756162984ecc44fbead19b809440eaeR313-R325